cumulusci/cli/project.py

```diff
@@ -210,10 +210,10 @@ def project_init(runtime):
             click.style("Minimum code coverage percentage", bold=True), default=75
         )
 
-    init_from_context(context)
+    init_from_context(context, echo=True)
 
 
-def init_from_context(context: Dict[str, object]):
+def init_from_context(context: Dict[str, object], echo: bool = False):
```

Comment:
It would be ideal if `context` used the same defaults that are embedded in the prompts above, so that it's not necessary to supply all the keys if you want to use the defaults for some of them. But this is fine as a first pass.